apps/08jquery-towers-of-hanoi/script.js

```diff
@@ -1,5 +1,33 @@
 'use strict';
+var $block = null;
 
 $(document).ready(function() {
-  // Put app logic here
+  $('div[data-stack]').on('click', function() {
+
+    if ($block === null) {
+      //picking up the bock
+      $block = $(this).children().last().detach();
+    } else {
+      //places block on new stack
+      if ($(this).children().length === 0 || $(this).children().last().data('block') > $block.data('block')) {
```

Comment:
An extra comment here about the two scenarios might help a reader understand a bit better